projects/plugins/jetpack/package.json

```diff
@@ -23,7 +23,7 @@
 		"build-extensions": "webpack --config ./tools/webpack.config.extensions.js",
 		"build-php": "composer install --ignore-platform-reqs",
 		"build-production": "pnpm run distclean && pnpm run install-if-deps-outdated && pnpm run build-production-client && pnpm run build-production-php && pnpm run build-production-extensions && pnpm run build-production-search && pnpm run build-asset-cdn-json",
-		"build-production-concurrently": "pnpm run distclean && pnpm run install-if-deps-outdated && pnpx concurrently 'pnpm:build-production-client' 'pnpm:build-production-php' 'NODE_ENV=production pnpm run build-extensions' 'NODE_ENV=production pnpm run build-search' && pnpm run build-asset-cdn-json",
+		"build-production-concurrently": "pnpm run distclean && pnpm run install-if-deps-outdated && pnpx concurrently 'pnpm:build-production-client' 'pnpm:build-production-php' 'NODE_ENV=production pnpm run build-extensions' && NODE_ENV=production pnpm run build-search && pnpm run build-asset-cdn-json",
```

Comment:
@jsnmoon reports:

" I think you could drop the production build change for this PR. That was originally needed when validate-es5 was around, but is now unnecessary."